app/api.py

```diff
@@ -81,8 +81,8 @@ async def update_mentees(profile_id: str, update_data: MenteeUpdate):
 
 
 @API.post("/match/{profile_id}")
-async def match(profile_id: str, n_matches: int):
-    return {"result": API.matcher(n_matches, profile_id)}
+async def match(profile_id: str, n_matches=5):
```

Comment:
Setting the default to 5 isn't exactly what Web asked for. What they requested is that all valid matching Mentors are returned if n_matches isn't provided. To get this behavior let's set the default here to None. So this code would look like the following:
```
async def match(profile_id: str, n_matches=None):
```
Switching the order of these parameters is ok, however if we had tests that checked our matcher algorithm, they would have needed to be updated as well. Just something to think about for future, refactoring can have a huge ripple effect.